source/api/presentation/3.0/index.md

```diff
@@ -363,6 +363,20 @@ Note that this is different to the `formats` property in the [Image API][image-a
 {"type": "Dataset", "format": "application/xml"}
 ```
 
+##### language
+The language or languages used in the content of an external resource. This property is already available from the Web Annotation model for content resources that are the body or target of an Annotation, however it _MAY_ also be used for resources referenced from `related`, `rendering`, `rights`, and `within`.
+
+The value _MUST_ be an array of strings.
+
+ * An external resource _MAY_ have one or more `language` values, and each value _MUST_ be a valid language code, as described under the [languages section][languages-prezi30].<br/>
+   Clients _SHOULD_ process the `language` of external resources to present the most appropriate content to the user.
+ * Other resource types _MUST NOT_ have a `language`.<br/>
+   Clients _SHOULD_ ignore `language` on other resource types.
+
```

Comment:
Why the `<br/>`s and not just separate bullets?

Reply:
Because it's the client requirements for that property on that resource type. 

It's also consistent all the way through section 3 (so would be another issue if there's a better way to manage it)

Reply:
OK. Fine with it as-is for now...I wonder if there's a better way to make that distinction.